.github/workflows/fly.yml

```diff
@@ -2,7 +2,7 @@ name: Fly Deploy
 on:
   push:
     branches:
-    - main
+    - Feat/deployment_setup
```

Comment:
should we change it to main so that only main branch deploys once changes get merged?